src/configupdater/parser.py

```diff
@@ -332,7 +432,7 @@ def _read(self, fp: Iterable[str], fpname: str):
             cur_indent_level = first_nonspace.start() if first_nonspace else 0
             if cursect is not None and optname and cur_indent_level > indent_level:
                 cursect[optname].append(value)
-                self.last_block.last_block.add_line(line)  # HOOK
+                self._add_option_line(line)  # HOOK
```

Comment:
I tried to clarify the intent of this line with a more meaningful name. I hope it is correct....

Reply:
Yes, thanks, it's a much better and easier to understand what's going on in the implementation of `_add_option_line`.